package.json

```diff
@@ -21,8 +21,6 @@
     "test:dev:ava": "ava",
     "test:ci:ava": "nyc -r lcovonly -r text -r json ava",
     "prepublishOnly": "run-s prepublishOnly:*",
-    "prepublishOnly:checkout": "git checkout master",
```

Comment:
We don't need those when publishing from CI.
Also I don't think they would work since the checkout action [checks out a specific commit](Only a single commit is fetched by default).

Reply:
Should we still keep those to keep the manual publishing process working?

Reply:
See https://github.com/netlify/build/pull/2442/commits/ed3775a9bbee66efaf890bfd0e79f79b952531c0